pkg/actors/actors_test.go

```diff
@@ -32,6 +31,31 @@ const (
 	TestKeyName = "key0"
 )
 
+// testRequest is the request object that encapsulates the `data` field of a request
+type testRequest struct {
+	Data interface{} `json:"data"`
+}
+
+type mockAppChannel struct {
```

Comment:
Why do we need this refactoring?

Reply:
You beat me before I could add the additional information :-)
See the details above.